apps/client/src/components/VisualStyles.js

```diff
@@ -7,6 +8,26 @@ export const FigCaption = styled.figcaption`
 export const Figure = styled.figure`
   min-height: 1px; /* IE11 Bug */
   margin: 0 0 24px 0;
+  ${({ isLoading, theme, animateLoading, backgroundImage }) =>
+    isLoading
+      ? css`
+          ${perceivedLoading(animateLoading)({ theme })}
+          height: 100px;
+          width: 100%;
+          & > * {
+            display: none;
+          }
+          &::before {
+            content: "";
+            display: block;
+            width: calc(100% - 30px);
+            height: 30px;
+            ${perceivedLoading(animateLoading)({ theme })}
+          }
+        `
+      : css`
+          background-image: url(${backgroundImage});
+        `}
```

Comment:
Shameless copy/paste from `ImageCardStyle` from asc-ui :-P

We don't have a background-image